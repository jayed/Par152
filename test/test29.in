 1      We the people of the United States,
 2      in order to form a more perfect union,
 3      establish justice,
 4      insure domestic tranquility,
 5      provide for the common defense,
 6      promote the general welfare,
 7      and secure the blessing of liberty
 8      to ourselves and our posterity,
 9      do ordain and establish the Constitution
10      of the United States of America.
