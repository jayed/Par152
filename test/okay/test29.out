 1      We the people of the United States, in order to form
 2  a more perfect union, establish justice, insure domestic
 3  tranquility, provide for the common defense, promote
 4  the general welfare, and secure the blessing of liberty
 5  to ourselves and our posterity, do ordain and establish
 6  the Constitution of the United States of America.
